styles/listingCard/listingCard.css

```diff
@@ -0,0 +1,15 @@
+.indent::before {
+  position: absolute;
+  top: -16px;
+  left: -14px;
+  content: '';
+  display: block;
+  width: 76px;
+  height: 26px;
+  background-position: right top;
+  background-repeat: no-repeat;
+  background-size: 76px auto;
+  background-image: url('../../public/indent_img.svg');
+  clip: rect(0px, 74px, 18px, 14px);
+  z-index: 1;
```

Comment:
skorzystaj z tailwindowego `@apply`:
https://tailwindcss.com/docs/functions-and-directives/#app
na pewno dla z-index:
https://tailwindcss.com/docs/z-index/#app:
position:
https://tailwindcss.com/docs/position/#app
i większości tych rzeczy oprócz niestandardowych width, heidht, top, left - na pewno masz klasy w tailwind

```css
.indent::before {
  @apply z-10;
  @apply absolute;
...
  top: -16px;
  left: -14px;
  content: '';
  width: 76px;
  height: 26px
}
```

Reply:
zrobiłem - mam nadzieję, że o to chodziło.